src/App.jsx

```diff
@@ -1,35 +1,35 @@
-import { useState } from 'react'
+import {useState} from 'react'
 import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
 
-  return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    const todoList = [
+        {
+            id: 1,
+            title: 'Complete CTD Hawk assignment'
+        },
+        {
+            id: 2,
+            title: 'Call vet'
+        },
+        {
+            id: 3,
+            title: 'Cook family dinner'
+        }
+    ];
+
+    return (
+        <>
+            <h1>Todo List</h1>
+            <ul>
+                {
+                    todoList.map((el) => <li id={el.id}>{el.title}</li>)
```

Comment:
Make sure you add the key attribute to this `<li>` element! If you right click on the webpage, then hit inspect and open the console tab in your devtools, you'll see this warning:

<img width="870" alt="image" src="https://github.com/christopher-ga/react-todo/assets/55410292/2677e2dd-0b9d-4819-8d79-093f01d3f6db">

Here's the link with a better explanation on why we need this: https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key